pyqtgraph/functions.py

```diff
@@ -372,6 +372,161 @@ def hsvColor(hue, sat=1.0, val=1.0, alpha=1.0):
     c = QtGui.QColor()
     c.setHsvF(hue, sat, val, alpha)
     return c
+    
+# Matrices and math taken from "CIELab Color Space" by Gernot Hoffmann
+# http://docs-hoffmann.de/cielab03022003.pdf
+MATRIX_XYZ_FROM_RGB = np.array( (
+    ( 0.4124, 0.3576, 0.1805),
+    ( 0.2126, 0.7152, 0.0722),
+    ( 0.0193, 0.1192, 0.9505) ) )
+    
+MATRIX_RGB_FROM_XYZ = np.array( (
+    ( 3.2410,-1.5374,-0.4985),
+    (-0.9692, 1.8760, 0.0416),
+    ( 0.0556,-0.2040, 1.0570) ) )
+
+VECTOR_XYZn = np.array( ( 0.9505, 1.0000, 1.0891) ) # white reference at illuminat D65
+
+def CIELabColor(L, a, b, alpha=1.0):
+    """
+    Generates as QColor from CIE L*a*b* values.
+    
+    Parameters
+    ----------
+    L: float
+        Lightness value ranging from 0 to 100
+    a, b: float
+        (green/red) and (blue/yellow) coordinates.
+    alpha: float, optional
+        Opacity, ranging from 0 to 1
+    """ 
```

Comment:
would you mind adding a [Notes](https://numpydoc.readthedocs.io/en/latest/format.html#notes) section to this docstring referencing where this "standard" or calculation comes from?

Reply:
Done. I also realized that the docstring needed manual import into the functions section of the docs.